airflow_spark_on_k8s_job_builder/core.py

```diff
@@ -277,7 +278,8 @@ def set_driver_cores(self, cores: int) -> "SparkK8sJobBuilder":
         if not cores:
             raise ValueError("Need to provide a non-empty value for the number of driver cores")
         self.get_job_params()["driver"]["cores"] = cores
-        [requested_cores, max_cores] = self._cast_cores_to_int(self.get_driver_cores(), self.get_driver_cores_limit(), "driver")
+        [requested_cores, max_cores] = self._cast_cores_to_int(self.get_driver_cores(), self.get_driver_cores_limit(),
```

Comment:
minor nit/suggestion: we usually prefer isolating formatting changes in dedicated commits (atomic commits story); it helps, for example, when you want to rollback a faulty commit, for example, as well as declaring intent for other developers ease of reviewability/debuging